pipenv/vendor/pip_shims/models.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         version,
         round_prereleases_up=True,
         base_import_path=None,
-        vendor_import_path="pip._vendor",
+        vendor_import_path="pipenv.patched.notpip._vendor",
```

Comment:
This should not get changed I don't think.   It would require a patch being its not an actual import that can be re-written, but I suspect it is not required because there is already a concept in `pip-shims` of specifying what the pip module is, so I think this line should not have to change.

Reply:
Fixed.